### DISCUSSION TITLE
Windows 11 upgrade

### DISCUSSION THREAD

**Discussion by HighlanderMisfits** [akira0245/MidiBard#48] [Q&A]
I'm considering making the big change to Windows 11, but I'm concerned that as soon as I make the change over I won't be able to play music!

Can you think of any issue I might run into when upgrading to windows 11?

**Comment by akira0245** [comment]
Sorry I'm have no idea/test if MidiBard will compatible with win11, I'm still using win10, probably others could give you more advice.
